contrib/initramfs/scripts/zfs.in

```diff
@@ -900,7 +900,7 @@ mountroot()
 	    }' |
 	    while read i
 	do
-		if [ -e "/sys/block/$i/queue/scheduler" ]
+		if grep -qs "/sys/block/$i/queue/scheduler" -e noop
```

Comment:
You can get rid of the `-e`: `grep -sq noop /sys/block/sda/queue/scheduler`

Reply:
I agree. Thanks